UI/Panels/InputConfigPanel.cs

```diff
@@ -698,9 +733,11 @@ private void inputsDataGridView_MouseDown(object sender, MouseEventArgs e)
         private void inputsDataGridView_MouseMove(object sender, MouseEventArgs e)
         {
             if (MouseButtons.Left == (e.Button & MouseButtons.Left))
-            {                
+            {
                 // When mouse leaves the rectangle, start drag and drop
-                if (RectangleMouseDown != Rectangle.Empty && !RectangleMouseDown.Contains(e.X, e.Y))
+                if (RectangleMouseDown != Rectangle.Empty &&
```

Comment:
We could change this expression so to do an early return

Additionally, we should provide a message to the user that dragging is not possible because sorting is active... users likely won't understand why suddenly the UI behaves differently.

Reply:
"We could change this expression so to do an early return."
What do you mean? After the if there is no more code. And I think it is evaluated step by step and if the first condition fails the others are not executed any more by .net.

Reply:
Ok in context with the change I make, I understand.

Reply:
Done.